public/locales/es/common.json

```
@@ -0,0 +1,272 @@
+{
+  "actions": {
+    "ok": "OK"
+  },
+  "balances": {
+    "apy": "APY",
+    "apyRange": "Rango de APY",
+    "balance": "Saldo",
+    "depositValue": "Cantidad depositada",
+    "earned": "Ganancias",
+    "marketCap": "Capitalización de mercado",
+    "max": "Max",
+    "pending": "Pendiente",
+    "picklePerBlock": "{{ amount }} PICKLEs son creados en cada bloque.",
+    "picklePerSecond": "{{ amount }} PICKLEs son creados cada segundo.",
+    "poolBalance": "Cantidad acumulada",
+    "poolSize": "Valor acumulado",
+    "poolSizeTooltip": "Total ETH/PICKLE pool value on Uniswap.",
+    "ratio": "Ratio",
```

@yabirgb dunno if this is better. I guess `ratio` could be considered a term of art.
```suggestion
    "ratio": "Relación",
```